uploader/uploader.py

```diff
@@ -30,6 +31,13 @@ def run(self):
             data = json.loads(self.body)
             file = os.path.join("/output", data['cdr_output'])
             logging.debug(f"Uploading data for {data['cog_id']} from {file}")
+            if not os.path.exists(file):
```

Comment:
To prevent all the if if if statements, additionally this makes it easy to try once twice, n times.
```suggestion
            counter = 0
            while not os.path.exists(file):
                counter = counter + 1
                if counter > 2: # maybe make a variable above
                    raise ValueError(f"File {file} does not exist for uploader!!!")  
                time.sleep(1)
```

Reply:
Ok, yeah, this makes sense.  I'll commit this, and then make the parameters (wait time and wait count) variables that are set in the function so they're easy to adjust.